File: tournaments/equationTemplate/equationTemplate.cs
bool equationTemplate(int[] values) {
    if (values[0] * values[1] == values[2] * values[3]) return true;
    if (values[0] * values[2] == values[1] * values[3]) return true;
    if (values[0] * values[3] == values[1] * values[2]) return true;
    if (values[0] * values[1] * values[2] == values[3]) return true;
    if (values[0] * values[1] * values[3] == values[2]) return true;
    if (values[0] * values[2] * values[3] == values[1]) return true;
    if (values[1] * values[2] * values[3] == values[0]) return true;
    return false;
}

File: tournaments/equationTemplate/equationTemplate.js
function equationTemplate(values) {
    const [a, b, c, d] = [...values];
    
    if (a * b === c * d) return true;
    if (a * c === b * d) return true;
    if (a * d === b * c) return true;
    if (a * b * c === d) return true;
    if (a * b * d === c) return true;
    if (a * c * d === b) return true;
    
    return b * c * d === a;
}
